std/traits.d

```diff
@@ -6553,3 +6554,70 @@ unittest
     static assert(getUDAs!(c, Attr)[1].name == "Pi");
     static assert(getUDAs!(c, Attr)[1].value == 3);
 }
+
+/**
+ * Gets all symbols within `symbol` that have the given user-defined attribute.
+ */
+template getSymbolsByUDA(alias symbol, alias attribute)
+{
+    import std.typetuple : Filter, staticMap, TypeTuple;
+    import std.traits : hasMember;
+
+    static enum hasSpecificUDA(alias S) = hasUDA!(S, attribute);
```

Comment:
Nonsense like `template instance I!(d) cannot use local 'd' as parameter to non-global template I(alias T)`